app/packages/core/src/plugins/SchemaIO/components/GridView.tsx

```diff
@@ -9,7 +9,7 @@ export default function GridView(props: ViewPropsType) {
   const { schema, path, data } = props;
   const { properties, view = {} } = schema as ObjectSchemaType;
   const { alignX, alignY, align_x, align_y, gap = 1, orientation } = view;
-  const direction = orientation === "vertical" ? "row" : "column";
+  const direction = orientation === "horizontal" ? "row" : "column";
```

Comment:
...so this was broken this whole time!?

Reply:
It did work but It's reversed for flex interestingly